src/main/webapp/mentors.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Get mentors from the mentor servlet.
+ */
+function getMentors() {
+  console.log("entering get mentors function/n");
+  let url = new URL('/mentors', location.protocol + '//' + location.hostname);
+
+  fetch(url).then(response => response.json()).then(mentors => {
+    console.log(mentors);
+    const mentorsContainer = document.getElementById('mentors-container');
+    mentorsContainer.innerHTML = '';
+    for (const mentor of mentors) {
+      mentorsContainer.appendChild(createMentorElement(mentor.name, mentor.description));
+    }
+  });
+}
+
+/**
+ * Creates a mentor container element from input text.
+ * @param {String} name
+ * @param {String} description
+ * @return {<div class="p-1 col-lg-4">} mentorContainer
```

Comment:
I'm not sure if this is valid JSDoc. Consider replacing with `Element`. For context take a look at the return type for `document.createElement()` [here](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement).

Reply:
Yeah, I think it should be ```HTMLElement```